test/e2e/framework/job/fixtures.go

```diff
@@ -54,7 +64,7 @@ func NewTestJob(behavior, name string, rPol v1.RestartPolicy, parallelism, compl
 						{
 							Name: "data",
 							VolumeSource: v1.VolumeSource{
-								EmptyDir: &v1.EmptyDirVolumeSource{},
+								HostPath: &v1.HostPathVolumeSource{Path: "/tmp"},
```

Comment:
this path should be scoped and made more narrow (e.g. `/tmp/job-e2e/$random-dir`), not expose all of /tmp to the pod (and leave the touch command below alone)

Reply:
also, should we only make this a HostPath volume if nodeName is set?

Reply:
Done